src/commands/toolingHub/liveKomaciAnalyze.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import * as vscode from 'vscode';
+import { commands, window, ExtensionContext } from 'vscode';
+import { CoreExtensionService } from '../../services/CoreExtensionService';
+
+const commandName = 'salesforcedx-vscode-offline-app.liveKomaciAnalysis';
+
+export class LiveKomaciAnalyzeCommand {
+    static async execute(
+        sourceUri: vscode.Uri | vscode.Uri[] | undefined,
+        uris: vscode.Uri[] | undefined
+    ) {
+        const telemetryService =
+            await CoreExtensionService.getTelemetryService();
```

Comment:
`CoreExtensionService.getTelemetryService()` is a synchronous function. No `await` necessary.